### PR TITLE
Document DaemonSet eviction opt in/out behavior

### PR DESCRIPTION
**Pull request by x13n**
/hold until https://github.com/kubernetes/autoscaler/pull/4172 is merged

### CODE REVIEW

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -434,6 +435,30 @@ spec:
...
+
+```
+"cluster-autoscaler.kubernetes.io/enable-ds-eviction": "true"
+```
+
+It is also possible to disable DaemonSet pods eviction expicitly:
+
+
+```
+"cluster-autoscaler.kubernetes.io/enable-ds-eviction": "true"
```
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md:453]: s/true/false

Comment (x13n) [reply cluster-autoscaler/FAQ.md]: Good catch, thanks!

**Pull request merged**